curricula/serializers.py

```diff
@@ -42,11 +42,12 @@ class LessonSerializer(serializers.ModelSerializer):
     student_resources = serializers.SerializerMethodField()
     vocab = serializers.SerializerMethodField()
     blocks = serializers.SerializerMethodField()
+    keywords = serializers.SerializerMethodField()
 
     class Meta:
         model = Lesson
         fields = ('title', 'number', 'student_desc', 'teacher_desc',
-                  'student_resources', 'teacher_resources', 'vocab', 'blocks')
+                  'student_resources', 'teacher_resources', 'vocab', 'blocks', 'keywords')
```

Comment:
I'm not sure what this file does currently but is there any potential risk to adding another field?

Reply:
I believe it'll start showing up on the API view: https://www.codecurricula.com/api/v1/curriculum/csf-1718/coursea/2/

@mrjoshida are these views used? Any downside to adding a few more attributes to the API view?

Reply:
I can't see any reason that'd be a problem